examples/pydantic-ai/agent.py

```
@@ -0,0 +1,81 @@
+from pydantic_ai import Agent, RunContext
+
+
+class WeatherResponse(BaseModel):
+    """Structured weather response"""
+
+    location: str
+    temperature: str
+    description: str
+
+
+def get_weather(ctx: RunContext, location: str) -> dict:
```

Ruff (0.11.9)

Unused function argument: `ctx`

(ARG001)